tidy3d/web/api/mode.py

```diff
@@ -231,6 +231,12 @@ def handle_mode_solver(index, progress, pbar):
 
     if verbose:
         console.log(f"[cyan]Running a batch of [deep_pink4]{num_mode_solvers} mode solvers.\n")
+
+        # Create the folder before running the parallel computation and wait one second before running the batch simulation job
+        folder = Folder.create(folder_name=folder_name)
+        import time
+        time.sleep(1)
```

Comment:
Put `from time import sleep` at the top level of this file and use `sleep(1)` here.

But separately, @magiWei , is there a better way to wait to make sure that the folder is created? This seems too hacky.

Reply:
The current way to create a folder is https synchronous interaction, the interface will respond to success or failure. Therefore, getting the interface returned will let you know the interface created by the folder.

Reply:
You mean there should be no need for a `time.sleep()` even in the current code? @prashkh do you know why you had to add it?